android/filament-android/src/main/java/com/google/android/filament/Renderer.java

```diff
@@ -44,6 +44,75 @@
 public class Renderer {
     private final Engine mEngine;
     private long mNativeObject;
+    private DisplayInfo mDisplayInfo;
+    private FrameRateOptions mFrameRateOptions;
+
+    /**
+     * Information about the display this renderer is associated to
+     */
+    public static class DisplayInfo {
+        /**
+         * Refresh rate of the display in Hz. Set to 0 for offscreen or turn off frame-pacing.
+         * On Android you can use {@link android.view.Display#getRefreshRate()}.
+         */
+        public float refreshRate = 60.0f;
+
+        /**
+         * How far in advance a buffer must be queued for presentation at a given time in ns
+         * On Android you can use {@link android.view.Display#getPresentationDeadlineNanos()}.
+         */
+        public long presentationDeadlineNanos = 0;
+
+        /**
+         * Offset by which vsyncSteadyClockTimeNano provided in beginFrame() is offset in ns
+         * On Android you can use {@link android.view.Display#getAppVsyncOffsetNanos()}.
+         */
+        public long vsyncOffsetNanos = 0;
+    };
+
+    /**
+     * Use FrameRateOptions to set the desired frame rate and control how quickly the system
+     * reacts to GPU load changes.
+     *
+     * interval: desired frame interval in multiple of the refresh period, set in DisplayInfo
+     *           (as 1 / DisplayInfo.refreshRate)
+     *
+     * The parameters below are relevant when some Views are using dynamic resolution scaling:
+     *
+     * headRoomRatio: additional headroom for the GPU as a ratio of the targetFrameTime.
```

Comment:
nit since this is an existing comment, but I would say percentage rather than ratio since ratios usually have the context of two quantities.

Reply:
(on second thought please ignore this nit, my other comments are more useful)